guides/common/assembly_using-foreman-webhooks.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-foreman-webhooks_{context}"]
+= Using Foreman Webhooks 
+
+A webhook in web development is a method of augmenting or altering the behavior of a web page or web application with custom callbacks. Webhooks are "user-defined HTTP callbacks". They are usually triggered by some event, such as pushing code to a repository or a comment being posted to a blog. When that event occurs, the source site makes an HTTP request to the URL configured for the webhook. Users can configure them to cause events on one site to invoke behavior on another. Because webhooks use HTTP, they can be integrated into web services without adding new infrastructure.
```

Comment:
This is taken straight from Wikipedia: https://en.wikipedia.org/wiki/Webhook which is fine but we need to give attribution somewhere. @melcorr I guess there is already a known process/place to put it?

Reply:
I would rather not borrow so obviously from Wikipedia. I'll  take a look at this initial paragraph. It's a bit wordy.

Reply:
Users need an introduction to the concept, why and how it is useful. @lzap added this to the google doc to give the writer a foundation. Suyog just copy pasted it rather than developing information.

Reply:
https://github.com/theforeman/foreman-documentation/pull/599 maybe that's a good place to start the discussion?

Reply:
A webhook is a way for a web page or web application to provide other applications with information. The information is provided immediately. Specific events trigger customized callbacks, such as sending an email when a user registers with a website, or acknowledging online purchases. Because webhooks use HTTP, no new infrastructure need be added to existing web services.

Just a suggestion.